app/main.py

```diff
@@ -1,8 +1,47 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str
+                 ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: int,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int
+                 ) -> None:
+        self.distance_from_city = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def wash_single_car(self, car: Car) -> None:
+        if car.clean_mark < self.clean_power:
+            car.clean_mark = self.clean_power
+
+    def calculate_washing_price(self, car: Car) -> float:
+        washing_price = 0
+        if car.clean_mark < self.clean_power:
+            washing_price = (car.comfort_class
+                             * (self.clean_power - car.clean_mark)
+                             * self.average_rating / self.distance_from_city)
+        return round(washing_price, 1)
+
+    def serve_cars(self, cars_list: list) -> float:
```

Comment:
Change your type annotation and variable name to indicate that you expect to obtain a list of `Car` class instances for `cars`, the `_list` part in the variable name is unnecessary here.
```suggestion
    def serve_cars(self, cars: list[Car]) -> float:
```